osu.Game/Overlays/DirectOverlay.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenTK;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Database;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Backgrounds;
+using osu.Game.Overlays.Direct;
+
+namespace osu.Game.Overlays
+{
+    public class DirectOverlay : WaveOverlayContainer

```

Comment:
It's assumed due to the way that it's displayed, it overlays the content below it instead of moving to a new screen.